pages/ksphere/konvoy/1.4/gpu/index.md

```diff
@@ -73,7 +73,7 @@ spec:
       nvidia: {}
   addons:
   - configRepository: https://github.com/mesosphere/kubernetes-base-addons
-    configVersion: stable-1.16-1.2.0
+    configVersion: testing-1.9.0-2
```

Comment:
We shouldn't merge this version

Reply:
okay let's fix that up in konvoy repo

Reply:
Ok, so I should NOT merge this PR?

Reply:
Its valid in the Konvoy repo because the default version is set to `testing-1.9.0-2`.
This brings up an issue though of updating docs separate from a release, this could happen where unintended changes are brought it.

Reply:
in general since this will be automated an on a schedule, we can reject a PR and wait for the next one once the fixes are in place.

> This brings up an issue though of updating docs separate from a release

for this though, we probably shouldn't put testing tags in the konvoy docs. We can update them on release days.

Side note, this PR also shows we updated docs on the docs site but not in Konvoy repo https://github.com/mesosphere/dcos-docs-site/pull/3128/files#diff-d0520c708a54a562f43af97cf631e535R58